tools/src/test/resources/dev/run-qualification-tests.sh

```diff
@@ -0,0 +1,451 @@
+#!/usr/bin/env bash
```

Comment:
sorry can we update the script name to something like generate-qualification-test-results, otherwise someone might think this is to run the tests